schema-sandbox/schema.json

```diff
@@ -64,6 +126,8750 @@
+                    "type": "array",
+                    "uniqueItems": true
+                },
+                "copyright": {
+                    "type": "string"
+                },
+                "data-type": {
+                    "type": "string"
+                },
+                "database": {
+                    "type": "string"
+                },
+                "database-provider": {
+                    "$ref": "#/definitions/entity"
+                },
+                "date-accessed": {
+                    "$ref": "#/definitions/date"
+                },
+                "date-downloaded": {
+                    "$ref": "#/definitions/date"
+                },
+                "date-published": {
+                    "$ref": "#/definitions/date"
+                },
+                "date-released": {
+                    "$ref": "#/definitions/date"
+                },
+                "department": {
+                    "type": "string"
+                },
+                "doi": {
+                    "$ref": "#/definitions/doi"
+                },
+                "identifiers": {
+                    "items": {
+                        "$ref": "#/definitions/identifier"
+                    },
+                    "minItems": 1,
+                    "type": "array",
+                    "uniqueItems": true
+                },
+                "edition": {
+                    "type": "string"
+                },
+                "editors": {
+                    "items": {
+                        "anyOf": [
+                            {"$ref": "#/definitions/person"},
+                            {"$ref": "#/definitions/entity"}
+                        ]
+                    },
+                    "minItems": 1,
+                    "type": "array",
+                    "uniqueItems": true
+
+                },
+                "editors-series": {
+                    "items": {
+                        "anyOf": [
+                            {"$ref": "#/definitions/person"},
+                            {"$ref": "#/definitions/entity"}
+                        ]
+                    },
+                    "minItems": 1,
+                    "type": "array",
+                    "uniqueItems": true
+                },
+                "end": {
+                    "type": "integer"
+                },
+                "entry": {
+                    "type": "string"
+                },
+                "filename": {
+                    "type": "string"
+                },
+                "format": {
+                    "type": "string"
+                },
+                "institution": {
+                    "$ref": "#/definitions/entity"
+                },
+                "isbn": {
+                    "pattern": "^(?:ISBN(?:-1[03])?:? )?(?=[0-9X]{10}$|(?=(?:[0-9]+[- ]){3})[- 0-9X]{13}$|97[89][0-9]{10}$|(?=(?:[0-9]+[- ]){4})[- 0-9]{17}$)(?:97[89][- ]?)?[0-9]{1,5}[- ]?[0-9]+[- ]?[0-9]+[- ]?[0-9X]$",
```

Comment:
In the future, we should consider simplifiying this one as well

Reply:
Agree, same as for `doi`, perhaps drop here, and also see [my comment here](https://github.com/citation-file-format/citation-file-format/pull/123#discussion_r632392685) re a lax + a strict schema.

Reply:
I've created issue #130 so that discussion is not blocking merge of this PR